tests/unit/test_form.py

```diff
@@ -136,7 +136,7 @@ def test_get_metadata(self):
         form_object = MultiDict([
             ('reported_with', u'desktop-reporter'),
             ('url', u'http://localhost:5000/issues/new'),
-            ('extra_labels', [u'type-stylo', u'type-webrender-enabled']),
+            ('extra_labels', u'type-webrender-enabled'),
```

Comment:
`extra_labels` is expected to be a **list**.

so that would be.
```
('extra_labels', [u'type-webrender-enabled'])
```

You can run the test locally by doing `nosetests -v`

Reply:
Ohh, right. Thanks @karlcow